includes/Block/Latest_Stories_Block.php

```diff
@@ -0,0 +1,652 @@
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	const BLOCK_NAME = 'web-stories/latest-stories';
+
+	/**
+	 * Current block's block attributes.
+	 *
+	 * @var array Block Attributes.
+	 */
+	protected $block_attributes = [];
+
+	/**
+	 * Initializes the Web Stories embed block.
+	 *
+	 * @since
+	 *
+	 * @return void
+	 */
+	public function init() {
+		$this->register_script( self::SCRIPT_HANDLE, [ self::STORY_PLAYER_HANDLE, Tracking::SCRIPT_HANDLE ] );
+		$this->register_style( self::SCRIPT_HANDLE, [ self::STORY_PLAYER_HANDLE ] );
+		wp_register_script( 'amp-carousel-script', 'https://cdn.ampproject.org/v0/amp-carousel-0.1.js', [], 'v0', true );
+
+		wp_register_style(
+			self::STYLE_HANDLE,
+			WEBSTORIES_PLUGIN_DIR_URL . 'includes/assets/latest-stories.css',
+			[],
+			'v0',
+			false
+		);
+
+		wp_localize_script(
+			self::SCRIPT_HANDLE,
+			'webStoriesLatestBlockSettings',
+			$this->get_script_settings()
+		);
+
+		// todo: use register_block_type_from_metadata() once generally available.
+		register_block_type(
+			self::BLOCK_NAME,
+			[
+				'attributes'      => [
+					'align'                => [
+						'type'    => 'string',
+						'default' => 'none',
+					],
+					'viewType'             => [
+						'type'    => 'string',
+						'default' => 'grid',
+					],
+					'numOfStories'         => [
+						'type'    => 'number',
+						'default' => 5,
+					],
+					'numOfColumns'         => [
+						'type'    => 'number',
+						'default' => 2,
+					],
+					'orderByValue'         => [
+						'type'    => 'string',
+						'default' => '',
+					],
+					'isShowingTitle'       => [
+						'type'    => 'boolean',
+						'default' => true,
+					],
+					'isShowingDate'        => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+					'isShowingAuthor'      => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+					'isShowingViewAll'     => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+					'viewAllLinkLabel'     => [
+						'type'    => 'string',
+						'default' => '',
+					],
+					'isShowingStoryPlayer' => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+					'carouselSettings'     => [
+						'type'    => 'object',
+						'default' => [
+							'autoplay' => false,
+							'delay'    => '',
+							'loop'     => false,
+						],
+					],
+					'authors'              => [
+						'type'    => 'array',
+						'default' => [],
+					],
+					'imageOnRight'         => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+					'isStyleSquared'       => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+				],
+				'render_callback' => [ $this, 'render_block' ],
+				'editor_script'   => self::SCRIPT_HANDLE,
+				'editor_style'    => self::SCRIPT_HANDLE,
+				'style'           => self::STYLE_HANDLE,
+			]
+		);
+	}
+
+	/**
+	 * Returns script settings.
+	 *
+	 * @since
```

Comment:
add more info about `@since`

Reply:
We are not sure when our changes will be released. Can this be done at later stage?